apisix/plugins/opentelemetry.lua

```diff
@@ -169,6 +169,13 @@ local schema = {
                 type = "string",
                 minLength = 1,
             }
+        },
+        additional_header_attributes = {
```

Comment:
Better to rename this conf to show it is used for prefix. Otherwise, people will ask why `additional_attributes` can use `http_header` but we have another conf called additional_header_attributes.